### PR TITLE
Minitoken tutorial docs

### PR DESCRIPTION
**Pull request by 3100**

### CODE REVIEW

File: docs/yui-ibc-solidity/minitoken/overview/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+We will create a smart contract that allows us to transfer tokens between two ledgers using IBC.</p><p>You will learn how to</p><ul><li>Use IBC to create and send packets between blockchains.</li><li>Create a basic token and send tokens to another blockchain.</li></ul><p>Here, we will use Solidity to describe the smart contract.
+The following ledgers are used:</p><ul><li>Hyperledger Besu</li><li>Ethereum</li></ul><p>If you want to know more about IBC, please refer to
+<a href="https://github.com/cosmos/ibc" target="_blank" rel="noopener noreferrer">cosmos/ibc</a>
+for more information about IBC.
+Also, the solidity implementation of IBC is
```
Comment (siburu) [inline docs/yui-ibc-solidity/minitoken/overview/index.html:18]: Is this sentence unnecessary? (or did you forget to delete it?)

**Comment by 3100** [comment]
- [ ] Because of the modifications on the source code side, the tutorial document needs to be slightly revised to explain the changes.
  - Generating sol code with solidity-protobuf

File: docsrcs/yui-ibc-solidity/docs/minitoken/contract.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+- mint: issue a new token for a given account
+- burn: burn your own token
+- transfer: Transfer your token to another account.
+
+It also has a state reference function:
+- balanceOf: to get the token balance of an account.
+
+The following states are also available:
+- balance: the balance of each account
```
Comment (bluele) [inline docsrcs/yui-ibc-solidity/docs/minitoken/contract.md:27]: `_balances`, not `balance` ?

File: docsrcs/yui-ibc-solidity/docs/minitoken/contract.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+but we will use a simple one here.
+
+While ICS-20 uses denomination to distinguish the source ledger,
+the MiniToken implemented here handles the issuer's ledger without distinction.
+
+## Basic Functions
+
+It has the following basic operational functions:
+
+- mint: issue a new token for a given account
```
Comment (bluele) [inline docsrcs/yui-ibc-solidity/docs/minitoken/contract.md:19]: I think these function names may be easier to read if you enclose them in ``.
e.g. `mint`, `burn`

File: docsrcs/yui-ibc-solidity/docs/minitoken/contract.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+---
+
+# Create a contract
+
+We will implement a token that can be transferred between two ledgers using IBC.
+
+There is a token standard called [ICS-20](https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer),
```
Comment (bluele) [inline docsrcs/yui-ibc-solidity/docs/minitoken/contract.md:9]: ICS-20 might be better described the token "transfer" standard, not just "token".
Therefore, we should modify the sentence "but we will use a simple one here" to "but we don't support the standard here"?

**Review by bluele — APPROVED** [review]
LGTM :+1:

File: docsrcs/yui-ibc-solidity/docs/minitoken/contract.md [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+We won't cover the description of modifier implementations such as `onlyOwner`,
+so please refer to the source code if you are interested.
+
+### burn
+
+Reduces tokens by the specified amount for the specified account.
+
+```solidity
+function burn(address account, uint256 amount) onlyOwner external {
+    _burn(account, amount);
```
Comment (siburu) [inline docsrcs/yui-ibc-solidity/docs/minitoken/contract.md:71]: Should it be wrapped by `require(...)` ?

**Pull request merged**